docs/source/pythonapi/cryolike.convert_particle_stacks.rst

```diff
@@ -1,45 +1,9 @@
-cryolike.convert\_particle\_stacks package
+cryolike.convert\_particle\_stacks
```

Comment:
I cut a lot of references to parts of the API we don't actually want to publicize, as well as some Sphinx commands that I don't think we need/want.

Probably easiest to review the API docs via the web front-end rather than the github diff view.